.dockerignore

```diff
@@ -0,0 +1,32 @@
+**/__pycache__
+**/.venv
+**/.classpath
+**/.dockerignore
+**/.env
+**/.git
+**/.gitignore
+**/.git-blame-ignore-revs
+**/.project
+**/.settings
+**/.toolstarget
+**/.vs
+**/.vscode
+**/.github
+**/*.*proj.user
+**/*.dbmdl
+**/*.jfm
+**/docs
+**/tests
+**/bin
+**/charts
+**/docker-compose*
+**/compose*
+**/Dockerfile*
+**/node_modules
+**/npm-debug.log
+**/obj
+**/secrets.dev.yaml
+**/values.dev.yaml
+AUTHORS.md
+LICENSE.txt
+README.md
```

Comment:
if you can please add:
**/.tox
**/env
**/venv

Reply:
Just to double check, I have `**/.env` and `**/.venv` but you also want versions without the leading `.`?

Reply:
correct

Reply:
done